ui-modules/logout/app/index.js

```diff
@@ -26,10 +26,11 @@ import brCore from 'brooklyn-ui-utils/br-core/br-core';
 import brInterstitialSpinner from 'brooklyn-ui-utils/interstitial-spinner/interstitial-spinner';
 import brooklynModuleLinks from 'brooklyn-ui-utils/module-links/module-links';
 import brooklynUserManagement from 'brooklyn-ui-utils/user-management/user-management';
+import brServerStatus from 'brooklyn-ui-utils/server-status/server-status';
 
 import mainState from './views/main/main.controller';
 
-angular.module('app', [ngAnimate, ngCookies, uiRouter, brCore, brInterstitialSpinner, brooklynModuleLinks, brooklynUserManagement, mainState])
+angular.module('app', [ngAnimate, ngCookies, uiRouter, brCore, brInterstitialSpinner, brServerStatus, brooklynModuleLinks, brooklynUserManagement, mainState])
```

Comment:
Why do we want the `brServerStatus` directive in there? I don't think it useful to have on the logout page

Reply:
it creates a repeating call to the server on a page in this module so we easily confirm whether it is logged out or not.